drivers/sensor/fxas21002/fxas21002_trigger.c

```diff
@@ -16,13 +16,13 @@ static void fxas21002_gpio_callback(const struct device *dev,
 {
 	struct fxas21002_data *data =
 		CONTAINER_OF(cb, struct fxas21002_data, gpio_cb);
+	const struct fxas21002_config *config = dev->config;
```

Comment:
`dev` is a GPIO device here, not a sensor device

Reply:
Nice catch! Should there be another `CONTAINER_OF` to get the sensor `dev` or is there a better solution?

Reply:
Just realised that's not even possible... Not sure why, but I'm reluctant to keep a reference to `int_gpio` in `struct fxas21002_data`.

Reply:
Done!